extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -58,6 +58,7 @@ export class PublishDatabaseDialog {
 	private serverName: string | undefined;
 	protected optionsButton: azdataType.ButtonComponent | undefined;
 	private publishOptionsDialog: PublishOptionsDialog | undefined;
+	public publishOptionsModified: boolean = false;
```

Comment:
what if there were options specified in a publish profile that got loaded?

Reply:
options from profile PR is not out yet!

But, we decide to consider the default options from profile if profile loaded (final design for profile under discussion) which means the Reset shouldn't be enabled while profile load right! as they can only reset to profile, toggle reset scenario will be same for profile options or default options!

correct me if I'm missing something! Thanks

Reply:
ok this is fine for now. Yes we still need to make a final decision on if reset should reset to the profile's loaded values or to the DacFx defaults